flink-table/flink-table-planner/src/main/java/org/apache/calcite/rel/logical/LogicalTableScan.java

```diff
@@ -103,7 +105,9 @@ public RelNode copy(RelTraitSet traitSet, List<RelNode> inputs) {
     // This file should be remove once CALCITE-4581 is fixed.
     @Override
     public RelWriter explainTerms(RelWriter pw) {
-        return super.explainTerms(pw).itemIf("hints", getHints(), !getHints().isEmpty());
+        List<RelHint> hintWithoutAlias = FlinkHints.getHintsWithoutAlias(getHints());
+        return super.explainTerms(pw)
+                .itemIf("hints", hintWithoutAlias, !hintWithoutAlias.isEmpty());
```

Comment:
Is it as expected that alias hint non-empty when alias is not declared? 
Too many plan changes because of this